### PR TITLE
Linode power updates

### LINKED ISSUE #582: Add delete to Linode action dropdown

### LINKED ISSUE #1452: Poll status on multiple objects using X-Filter
polling for status changes across many objects should be done with a single request and multiple labels/ids in the X-Filter header

### LINKED ISSUE #1606: Linode power actions maybe should have confirm

### LINKED ISSUE #1740: Linode mass actions don't trigger status dropdown

### LINKED ISSUE #1939: provide console launching regardless of linode power on linode list items

### LINKED ISSUE #1940: add a confirm modal to mass action linode reboot

### LINKED ISSUE #1947: make entire button open the dropdowns on primary list items

### LINKED ISSUE #1986: Rebuild doesn't trigger StatusDropdown polling

### LINKED ISSUE #1989: Provisioning stays at 100% after provisioning is done until a page transition is made.

### PR DESCRIPTION
**Pull request by eatonphil**
* Add delete to Linode action dropdown, closes #582 
* Linode power actions should have confirm, closes #1940, closes #1606 
  * All but power on, which shouldn't need a confirm.
* Allow you to launch lish no matter the power state, closes #1939 
* Rebuild triggers statusdropdown polling, closes #1986
* Polls on mass events, closes #1740
* Updates Polling object to allow one request for all polling ids, closes #1452 
* Adds default first button action that opens dropdown, closes #1947
* Changes behavior enough relevant to #1989 such that I cannot reproduce it, closes #1989

### CODE REVIEW

**Comment by na3d** [comment]
Linter Error
> src/linodes/components/StatusDropdown.js
> 50:5  warning  Unexpected console statement  no-console

File: src/api/polling.js [hunk 1]
```diff
@@ -65,9 +68,18 @@ export default function Polling(args) {
     }
   }
 
+  function isPolling(id) {
```
Comment (mparke) [inline src/api/polling.js]: nice

**Comment by mparke** [comment]
reboot confirm doesn't trigger status animation?

**Comment by eatonphil** [comment]
I moved the progress logic out of the API coded and implemented a separate progress bar so you can still see what the Linode is doing while it's transitioning.
screen shot 2017-06-01 at 8 06 59 pm

**Comment by eatonphil** [comment]
Fixes selected Linode background color too. We probably want a different color though...
screen shot 2017-06-01 at 8 21 20 pm

**Comment by mparke** [comment]
For later: one thing we may want to try and improve further is the rebuild status change, it goes from powering down > offline ( which is cool ), but then nothing else until the events notification comes up that it was rebuilt.

**Pull request merged**